src/crudeditor-lib/views/edit/reducer.js

```diff
@@ -315,6 +308,27 @@ export default modelDefinition => (
   // ███████████████████████████████████████████████████████████████████████████████████████████████████████
   } else if (type === TAB_SELECT) {
     const { tabName } = payload; // may be falsy, i.e. not specified.
+
+    // reset formInstance to be equal to persistentInstance
+    if (!isEqual(storeState.formInstance, storeState.persistentInstance)) {
+      newStoreStateSlice.formInstance = storeState.persistentInstance
+    }
+
+    newStoreStateSlice.formatedInstance = Object.keys(storeState.persistentInstance).reduce(
+      (rez, fieldName) => {
+        const fieldLayout = findFieldLayout(fieldName)(storeState.formLayout);
+        return fieldLayout ? {
+          ...rez,
+          [fieldName]: formatField({
+            value: storeState.persistentInstance[fieldName],
+            type: modelDefinition.model.fields[fieldName].type,
+            targetType: fieldLayout.render.valueProp.type
+          })
+        } : rez; // Field from the modelDefinition.model.fields is not in formLayout => it isn't displayed in Edit View.
+      },
+      {}
+    );
+
```

Comment:
all errors must be reset  => declare a function syncronizeInstances() which syncronizes formInstance and formatedInstance to persistentInstance.  Use the function both from INSTANCE_EDIT_SUCCESS/INSTANCE_SAVE_SUCCESS and TAB_SELECT action handlers.

```javascript
function syncronizeInstances({
persistentInstance = storeState.persistentInstance,
formLayout = storeState.formLayout
}) {
    newStoreStateSlice.formInstance = u.constant(cloneDeep(persistentInstance));

    newStoreStateSlice.formatedInstance = u.constant(Object.keys(persistentInstance).reduce(
      (rez, fieldName) => {
        const fieldLayout = findFieldLayout(fieldName)(formLayout);
        return fieldLayout ? {
          ...rez,
          [fieldName]: formatField({
            value: instance[fieldName],
            type: modelDefinition.model.fields[fieldName].type,
            targetType: fieldLayout.render.valueProp.type
          })
        } : rez; // Field from the modelDefinition.model.fields is not in formLayout => it isn't displayed in Edit View.
      },
      {}
    ));

    newStoreStateSlice.errors = u.constant({
      fields: Object.keys(persistentInstance).reduce(
        (rez, fieldName) => ({
          ...rez,
          [fieldName]: []
        }),
        {}
      )
    });
```